wordpress.org/public_html/wp-content/plugins/wporg-gp-custom-errors/wporg-gp-custom-errors.php

```diff
@@ -186,7 +203,81 @@ public function error_timezone_date_format( $original, $translation, $gp_origina
 			return esc_html__( 'The translation has empty spaces, new lines or another similar elements.', 'glotpress' );
 		}
 
-		return esc_html__( 'Must be a valid timezone date format.', 'glotpress' );
+		return esc_html__( 'Must be a valid date format.', 'glotpress' );
+	}
+
+	/**
+	 * Adds an error for text direction.
+	 *
+	 * @param string      $original    The original string.
+	 * @param string      $translation The translated string.
+	 * @param GP_Original $gp_original The GP_original object.
+	 * @param GP_Locale   $locale      The locale.
+	 * @return string|true The error message or true if no error.
+	 */
+	public function error_check_enum_list( $original, $translation, $gp_original, $locale ) {
+		if ( ! $this->is_core_project( $gp_original ) ) {
+			return true;
+		}
+		if ( is_null( $gp_original->context ) ) {
+			return true;
+		}
+		if ( !isset(self::CONTEXT_ENUM_CHECK_LIST[$gp_original->context])) {
+			return true;
+		}
+		if ( in_array( $translation, self::CONTEXT_ENUM_CHECK_LIST[$gp_original->context]) ) {
+			return true;
+		}
+
+		return esc_html__(
+			'Must be ' . $this->array_to_string( self::CONTEXT_ENUM_CHECK_LIST[$gp_original->context] ),
+			'glotpress'
+		);
```

Comment:
This should be a literal string, given it's being translated:

```suggestion
		return sptintnt(
			esc_html__(
				'Must be one of %s',
				'glotpress'
			),
			 $this->array_to_string( self::CONTEXT_ENUM_CHECK_LIST[$gp_original->context] )
		 );
```